recipe/meta.yaml

```diff
@@ -8,70 +8,70 @@ package:
+      - patches/0003-Inherit-CGO_-environment-variables.patch
+      - patches/0004-Add-conda-aware-default-values-for-GOPATH-and-GOBIN.patch
+      - patches/0005-cmd-link-internal-ld-disable-testWindowsBuildmodeCSh.patch
+      - patches/0006-issue10607-is-only-valid-when-running-with-CGO.patch
+      # https://github.com/conda-forge/go-feedstock/pull/150#issuecomment-1153458502
+      - patches/0007-Revert-cmd-link-enable-internal-linker-in-more-cases.patch
+  # Update this with a release from https://go.dev/dl/
+  - folder: go-bootstrap  # [aarch64 or ppc64le or osx or win64 or (linux and x86_64)]
+    url: https://go.dev/dl/go{{ version }}.linux-arm64.tar.gz  # [aarch64]
+    sha256: 49960821948b9c6b14041430890eccee58c76b52e2dbaafce971c3c38d43df9f  # [aarch64]
+
+    url: https://go.dev/dl/go{{ version }}.linux-amd64.tar.gz  # [linux and x86_64]
+    sha256: acc512fbab4f716a8f97a8b3fbaa9ddd39606a28be6c2515ef7c6c6311acffde  # [linux and x86_64]
+
+    url: https://go.dev/dl/go{{ version }}.linux-ppc64le.tar.gz  # [ppc64le]
+    sha256: 4137984aa353de9c5ec1bd8fb3cd00a0624b75eafa3d4ec13d2f3f48261dba2e  # [ppc64le]
+
+    url: https://go.dev/dl/go{{ version }}.windows-amd64.zip  # [win64]
+    sha256: b33584c1d93b0e9c783de876b7aa99d3018bdeccd396aeb6d516a74e9d88d55f  # [win64]
+
+    url: https://go.dev/dl/go{{ version }}.darwin-arm64.tar.gz  # [osx and arm64]
+    sha256: e46aecce83a9289be16ce4ba9b8478a5b89b8aa0230171d5c6adbc0c66640548  # [osx and arm64]
+
+    url: https://go.dev/dl/go{{ version }}.darwin-amd64.tar.gz  # [osx and x86_64]
+    sha256: b2828a2b05f0d2169afc74c11ed010775bf7cf0061822b275697b2f470495fb7  # [osx and x86_64]
 
 build:
-  skip: True    # [win32 or linux32]
   binary_relocation: false
   detect_binary_files_with_prefix: false
-  force_ignore_keys:    # [win]
+  force_ignore_keys:   # [win]
     - c_compiler        # [win]
     - cxx_compiler      # [win]
     - fortran_compiler  # [win]
   # test data links to these DSOs
   missing_dso_whitelist:
-    - '**/libc.so.6'               # [linux]
+    - $RPATH/libc.so.6             # [linux and not cgo]
     - /usr/lib/libSystem.B.dylib   # [osx]
     - $SYSROOT\System32\winmm.dll  # [win]
   number: 0
+  skip: True  # [linux and s390x]
 
 requirements:
-  build:
-    - patch     # [unix]
-    - m2-patch  # [win]
   run:
     - {{ pin_subpackage(name, exact=true) }}
-    - {{ compiler('c') }}         # [unix and cgo]
-    - {{ compiler('m2w64_c') }}   # [win and cgo]
+    - {{ compiler('c') }}  # [unix and cgo]
+    - {{ compiler('m2w64_c') }}  # [win and cgo]
 
 test:
   commands:
     - go help
 
 outputs:
   - name: {{ name }}
-    script: {{ go_variant_str }}/build.sh  # [unix]
+    script: {{ go_variant_str }}/build.sh   # [unix]
     script: {{ go_variant_str }}/build.bat  # [win]
     requirements:
       build:
+        - {{ compiler('go14') }}  # [not (aarch64 or ppc64le or osx or win64 or (linux and x86_64))]
```

Comment:
If this is not used anymore, can this be removed?

Reply:
right, we should do ... this long selector actually means just s390x, which isn't supported here anyway